src/js/components/RangeSelector/doc.js

```diff
@@ -72,8 +72,37 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
-    description: 'The color foe the edge controls',
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
+    type: 'string',
+    defaultValue: 'accent-1',
+  },
+  'global.edgeSize.small': {
+    description: 'The possible sizes for the margin, padding and gap',
+    type: 'string',
+    defaultValue: '6px',
+  },
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'string',
+    defaultValue: `{
+      xsmall: '1px',
+      small: '2px',
+      medium: '4px',
+      large: '12px',
+      xlarge: '24px,
+    }`,
+  },
+  'global.colors.border': {
+    description: 'The color for the border',
+    type: 'string',
+    defaultValue: {
+      dark: 'rgba(255, 255, 255, 0.33)',
+      light: 'rgba(0, 0, 0, 0.33)',
+    },
+  },
+  'global.colors.control': {
+    description: 'The color for the edge controls',
     type: 'string | { dark: undefined, light: undefined }',
```

Comment:
Can you please edit this line to `type: 'string | { dark: string, light: string }'`?